packages/cache-persist/package.json

```diff
@@ -33,10 +33,10 @@
     "test": "jest --coverage"
   },
   "dependencies": {
-    "idb": "^4.0.0"
+    "idb": "^4.0.0",
+    "@react-native-async-storage/async-storage": "1.13.3"
```

Comment:
I prefer it to be handled as devDependencies as the package is web-first 👍

Reply:
Makes sense! Will rectify and reinstate the lint comment! Thanks for explaining!